expenses.txt

```diff
@@ -1,8 +1,2 @@
-f | 0.6 | A | 2020-11-09
-a | 0.1 | SGD | 2020-11-08
-e | 0.5 | SGD | 2020-11-08
-b | 0.2 | USD | 2020-11-08
-b | 3.0 | SGD | 2020-11-07
-a | 1.0 | SGD | 2020-11-07
-c | 0.3 | SGD | 2020-11-07
-d | 0.4 | USD | 2020-11-07
+dinner | 0.6 | A | 2020-11-09
+tablet | 0.4 | USD | 2020-11-07
```

Comment:
This can put under .gitignore so it wont be uploaded

Reply:
Ah yes I think Zeyu did it in her PR.